ui/apps/platform/src/Containers/Compliance/Dashboard/ManageStandardsModal.tsx

```diff
@@ -39,21 +40,26 @@ function ManageStandardsModal({ standards, onSave, onCancel }): ReactElement {
                 .filter(({ hideScanResults, id }) => !hideScanResults !== showScanResultsMap[id])
                 .map(({ id }) => patchComplianceStandard(id, !showScanResultsMap[id]));
 
-            // TODO rewrite with ES2020 allSettled to solve async problem with all.
-            // TODO decide how to display results and update Formik state
-            // if some requests fail but other requests succeed.
-            Promise.all(patchRequestPromises)
-                .then(() => {
-                    fetchComplianceStandardsSortedByName()
-                        .then((standardsFetchedAfterPatchRequests) => {
-                            onSave(standardsFetchedAfterPatchRequests);
-                        })
-                        .catch((error) => {
-                            setErrorMessage(getAxiosErrorMessage(error));
-                        })
-                        .finally(() => {
-                            setSubmitting(false);
-                        });
+            Promise.allSettled(patchRequestPromises)
+                .then((results) => {
+                    let nRejected = 0;
```

Comment:
nit: My first Motorola assembler program only considered the first 6 characters of a Symbol unique, but we don't have that limitation anymore. 😉 

```suggestion
                    let numRejected = 0;
```